book/src/http_beacon.md

```diff
@@ -7,103 +7,125 @@ beacon chain and also historical information about beacon blocks and states.
+```
+{
+    attestation_1: {
+        attesting_indices: [u64],
+        data: {
+            slot: Slot,
+            index: u64,
+            beacon_block_root: Bytes32,
+            source: {
+                epoch: Epoch,
+                root: Bytes32
+            },
+            target: {
+                epoch: Epoch,
+                root: Bytes32
+            }
+        }
+        signature: Bytes32
     },
-    {
-        "beacon_block_root": "0x41ed5b253c4fc841cba8a6d44acbe101866bc674c3cfa3c4e9f7388f465aa15b",
-        "beacon_block_slot": 38375
+    attestation_2: {
+        attesting_indices: [u64],
+        data: {
+            slot: Slot,
+            index: u64,
+            beacon_block_root: Bytes32,
+            source: {
+                epoch: Epoch,
+                root: Bytes32
+            },
+            target: {
+                epoch: Epoch,
+                root: Bytes32
+            }
+        }
+        signature: Bytes32
     }
-]
+}
 ```
 
-## `/beacon/block_root`
-
-Returns the block root for the given slot in the canonical chain. If there
-is a re-org, the same slot may return a different root.
-
-### HTTP Specification
-
-| Property | Specification |
-| --- |--- |
-Path | `/beacon/block_root`
-Method | GET
-JSON Encoding | Object
-Query Parameters | `slot`
-Typical Responses | 200, 404
+### Returns
 
-## Parameters
+Returns `true` if the attester slashing was inserted successfully, or the corresponding error if it failed.
 
-- `slot` (`Slot`): the slot to be resolved to a root.
+### Example
 
-### Example Response
+### Request Body
 
 ```json
-"0xc35ddf4e71c31774e0594bd7eb32dfe50b54dbc40abd594944254b4ec8895196"
+{
+	"attestation_1": {
+		"attesting_indices": [0],
+		"data": {
+			"slot": 1,
+			"index": 0,
+			"beacon_block_root": "0x0000000000000000000000000000000000000000000000000100000000000000",
+			"source": {
+				"epoch": 1,
+				"root": "0x0000000000000000000000000000000000000000000000000100000000000000" 
+			},
+			"target": {
+				"epoch": 1,
+				"root": "0x0000000000000000000000000000000000000000000000000100000000000000" 
+			}
+		},
+		"signature": "0xb47f7397cd944b8d5856a13352166bbe74c85625a45b14b7347fc2c9f6f6f82acee674c65bc9ceb576fcf78387a6731c0b0eb3f8371c70db2da4e7f5dfbc451730c159d67263d3db56b6d0e009e4287a8ba3efcacac30b3ae3447e89dc71b5b9"
+	},
+	"attestation_2": {
+		"attesting_indices": [0],
+		"data": {
+			"slot": 1,
+			"index": 0,
+			"beacon_block_root": "0x0000000000000000000000000000000000000000000000000100000000000000",
+			"source": {
+				"epoch": 1,
+				"root": "0x0000000000000000000000000000000000000000000000000100000000000000"
+			},
+			"target": {
+				"epoch": 1,
+				"root": "0x0000000000000000000000000000000000000000000000000200000000000000"
+			}
+		},
+		"signature": "0x93fef587a63acf72aaf8df627718fd43cb268035764071f802ffb4370a2969d226595cc650f4c0bf2291ae0c0a41fcac1700f318603d75d34bcb4b9f4a8368f61eeea0e1f5d969d92d5073ba5fbadec102b45ec87d418d25168d2e3c74b9fcbb"
+	}
+}
```

Comment:
The attester_slashing inserted here will be valid (return `true`) but will make the beacon client behave strangely due to some invalid data being inserted (stopping block production since block header won't be valid anymore...). Should we find a way to explain it?